packages/design/tailwind/css/layout.css

```diff
@@ -26,4 +26,12 @@
   @apply gi-w-full;
 }
 
+.gi-layout-container-2-column {
```

Comment:
I'm not completely sure, but I think you could use the Stack component for that instead of creating column classes?

https://storybook-react.design-system.blocks.gov.ie/?path=/docs/layout-stack--docs

Reply:
The stack is available only on React.